docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -716,116 +686,312 @@ Where ``validateTake`` is defined as
 
 {% endif %}
 
+State
```

Comment:
Delete this "State" header - the real State header is below (at lines 863-864).

Reply:
Sounds good.